solidity/test/FullyBackedBondingTest.js

```diff
@@ -0,0 +1,451 @@
+const {accounts, contract, web3} = require("@openzeppelin/test-environment")
```

Comment:
I am getting test failures locally from time to time:
```
  1) FullyBackedBonding
       withdraw
         cannot be called before delegation lock period passes:
     AssertionError: Expected an exception but none was received
      at expectException (node_modules/@openzeppelin/test-helpers/src/expectRevert.js:25:10)
      at processTicksAndRejections (internal/process/task_queues.js:86:5)

  2) FullyBackedBonding
       isInitialized
         returns false when authorized but initialization period not passed yet:

      AssertionError: expected true to be false
      + expected - actual

      -true
      +false
      
      at Context.it (test/FullyBackedBondingTest.js:399:14)
      at processTicksAndRejections (internal/process/task_queues.js:86:5)
```

I think we need to set a little safer time margin for those tests.

Reply:
Should be fine now.